src/main/java/com/hotels/mutantswarm/MutantSwarmCore.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.mutantswarm;
+
+import java.util.ArrayList;
+import java.util.List;
+import com.hotels.mutantswarm.exec.MutantState;
+import com.hotels.mutantswarm.exec.MutatedSourceFactory;
+import com.hotels.mutantswarm.exec.SwarmResults;
+import com.hotels.mutantswarm.exec.MutatedSourceFactory.MutatedSource;
+import com.hotels.mutantswarm.exec.SwarmResults.SwarmResultsBuilder;
+import com.hotels.mutantswarm.model.MutantSwarmScript;
+import com.hotels.mutantswarm.model.MutantSwarmSource;
+import com.hotels.mutantswarm.model.MutantSwarmStatement;
+import com.hotels.mutantswarm.mutate.Mutation;
+import com.hotels.mutantswarm.plan.CompositeMutantFactory;
+import com.hotels.mutantswarm.plan.Mutant;
+import com.hotels.mutantswarm.plan.Swarm;
+import com.hotels.mutantswarm.plan.Swarm.SwarmFactory;
+import com.klarna.hiverunner.builder.Script;
+import com.klarna.hiverunner.builder.Statement;
+import com.klarna.hiverunner.sql.cli.CommandShellEmulator;
+import com.klarna.hiverunner.sql.split.StatementSplitter;
+import org.slf4j.Logger;
+
+public class MutantSwarmCore {
+
+  public MutantSwarmSource setUpScripts(
+      List<? extends Script> scriptsUnderTest,
+      Logger log,
```

Comment:
Don't pass the logger in as it will output the name of the class it was created in, not this class - this will be confusing in the log output. Instead just have this class create its own static logger.